README.md

```diff
@@ -7,28 +7,28 @@
 
 
 # message-io
-`message-io` is an event-driven message library to build network applications **easy** and **fast**.
-The library handles the internal OS socket in order to offer a simple event message API to the user.
+`message-io` is an event-driven message library to build network applications **quickly** and **easily**.
```

Comment:
My intention here with "easy" and "fast" is to indicate that message-io **is** easy and fast. Could "quickly" and "easily" mean something it offers instead of something it is?

Maybe what I want to say is: message-io is an easy and fast event-driven message library to build network applications. 

How do you think?

Reply:
Hmm -- "fast" definitely works, but "easy" typically refers to an action rather than a noun. Instead of "easy" I would say "easy-to-use". I think this also falls under the adjective order rules, so it would be "fast and easy-to-use" instead of the other way around. So the final result would be:

"`message-io` is a fast and easy-to-use event-driven message library for building network applications."

Reply:
Thanks! I like how it sounds